src/components/ActionDot.icon.js

```diff
@@ -0,0 +1,32 @@
+/**
```

Comment:
This is also being added in the `add icons` [PR](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/453)

Should you change the base branch?